content/tools/local-climate-change-snapshot/indicators.js

```diff
@@ -0,0 +1,143 @@
+  },
+  {
+    id: 'cddm',
+    categoryId: 'precipitation',
+    name: 'Maximum Length of Dry Spell',
+    units: 'days',
+    decimals: 0,
+    yScaleBaseValue: 0,
+    tagline: `The maximum length of dry spell for each year. In other words, 
+    the maximum number of consecutive days with precipitation < 1mm for each year.`,
+  },
+  {
+    id: 'speiltm1',
+    categoryId: 'precipitation',
+    name: 'SPEI 1-month',
+    units: 'months',
+    decimals: 1,
+    yScaleBaseValue: 0,
+    tagline: `Number of months in a year with a Standardised Precipitation-Evapotranspiration 
+    Index (SPEI) <= -1. SPEI is a multi-scalar drought index and can be used to detect, monitor 
+    and analyze droughts.`,
+    note: `The standardized precipitation-evaporation index (SPEI) depicts the combined 
+    impacts of precipitation deficits and potential evapotranspiration on soil moisture. 
+    SPEI does not include impacts from effects like wind speed, relative humidity or solar 
+    radiation impacts (typically short-term forcing) – making it more reflective of 
+    long-term hydrological and ecological drought conditions. Here we present SPEI 
+    calculated for a 9-month period, attempting to reflect a length slightly longer than 
+    California’s typical wet period.  A value less than -1 implies the drought is at 
+    least moderate in intensity, with more negative values representing more severe droughts.`,
+  },
+  {
+    id: 'swe',
+    categoryId: 'precipitation',
+    name: 'April SWE',
+    units: 'inches',
+    decimals: 1,
+    yScaleBaseValue: null,
+    tagline: `Snow Water Equivalent (SWE), is a commonly used measurement used by 
+    hydrologists and water managers to gage the amount of liquid water contained within 
+    the snowpack.`,
+  },
+  {
+    id: 'pr',
+    categoryId: 'precipitation',
+    name: 'Annual Precipitation',
+    units: 'inches',
+    decimals: 1,
+    yScaleBaseValue: null,
+    tagline: 'Total precipitation projected for a year',
+  },
+  {
+    id: 'fire',
+    categoryId: 'wildfire',
+    name: 'Annual Average Area Burned',
+    units: 'acres',
+    decimals: 1,
+    yScaleBaseValue: null,
+    tagline: 'Average of the area projected to be at risk to burning in a year.',
+  },
+  {
+    id: 'kbdigt600',
+    categoryId: 'wildfire',
+    name: 'KBDI > 600',
+    units: 'days',
+    decimals: 0,
+    yScaleBaseValue: null,
+    tagline: `Number of days in a year where Keetch-Byram Drought Index (KBDI) > 600.
+    KBDI provides an estimate for how dry the soil and vegetative detritus is.`,
+    note: `KBDI is cumulative. The KBDI values increase on dry and warm days and decrease 
+    during rainy periods. In California we would expect KBDI to increase from the 
+    end of the wet season (spring) into the dry season (summer & fall). The list 
+    below explains what values of KBDI represent:
+    <table class="table table-bordered table-sm mt-2" style="font-size:90%;width:auto;color:#616161;">
```

Comment:
I wonder if there's a way we could avoid stashing this table in a template literal? We can come back to it later.